en/docs/learn/login-to-identity-server-using-another-identity-server-oauth2.md

```diff
@@ -4,13 +4,12 @@ This tutorial guides you through setting up two instances of WSO2
 Identity Server and using one Identity Server to authenticate the user
 and login to the other one.
 
-1.  Start 2 Identity Servers. Start the 2nd Identity Server with its hostname set as "wso2is" with a relevant certificate added to the primary IS client-truststore and a port
+1.  Start two Identity Servers. Start the second Identity Server with its hostname as `wso2is` with a relevant certificate added to the primary IS client-truststore, and a port
```

Comment:
Start two WSO2 IS instances. Start the second WSO2 IS instance with `wso2is` configured as its hostname. Add the relevant certificate to the primary IS client truststore and set a port offset of 1. 

can break into two steps?